### PR TITLE
RQ docs

### PR DESCRIPTION
**Pull request by Harriethw**
### Checklist

* [x] Has this branch been rebased on top of the current `main` branch?

  <details>
  <summary>Explanation</summary>
  
  The branch should not be stale or have conflicts at the time reviews are requested.
  
  </details>

* [x] Is the CircleCI build passing?

### General points

<details>
<summary><strong>Other things to check</strong></summary><p></p>

* Make sure `fixtures/test_data.yaml` is maintained when updating models
* Consider the admin site when making changes to models
* Use select-/prefetch-related field lists in views and search apps, and update them when fields are added
* Make sure the README is updated e.g. when adding new environment variables

</details>

See [docs/CONTRIBUTING.md](https://github.com/uktrade/data-hub-api/blob/main/docs/CONTRIBUTING.md) for more guidelines.

### CODE REVIEW

File: docs/Using RQ and RQScheduler.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+- [DataHubScheduler](https://github.com/uktrade/data-hub-api/blob/main/datahub/core/queues/scheduler.py)
+    - This is the class to ensure that we start workers and enqueue or schedule jobs in a consistent way.
+
+## How to schedule a job
+
+First decide if it's a one off job, or you want to it to repeat regularly.
+
+### enqueue 
+(schedule to run when it reaches the front of a queue)
+1. call `job_scheduler()` with the function you want to get called, e.g. `job_scheduler(hello_world)` some more [examples are in the tests](https://github.com/uktrade/data-hub-api/blob/main/datahub/core/test/queues/test_job_scheduler.py)
```
Comment (vfarah-if) [inline docs/Using RQ and RQScheduler.md:29]: Call ...

File: docs/Using RQ and RQScheduler.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+The core service for facilitating RQ exported information is [RQ exporter](https://github.com/mdawar/rq-exporter). This will run on localhost:9726 to facilitate any RQ monitoring as data source. The configuration for the environment variables can be overridden, see more information at [Environment variable configuration](https://github.com/mdawar/rq-exporter#configuration). The datasource will be setup through _Prometheus_.
+
+### Environment
+
+**NOTE**: If you have a new or flushed docker system make sure you run `docker-compose -f docker-compose.yml -f docker-compose-rq-monitor.yml up --build` to build all the images or simply run `docker compose up --build` to build the core dependencies.
+
+- Starting all the services `make start-rq-mon`
+- Stopping all the services `make stop-rq-mon`
+
```
Comment (vfarah-if) [inline docs/Using RQ and RQScheduler.md:49]: Describe the services that get started and how you can access them and what they benefit

File: docs/Using RQ and RQScheduler.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+## Troubleshooting
+
+Troubleshooting
+1. Jobs are being queued but not started
```
Comment (vfarah-if) [inline docs/Using RQ and RQScheduler.md:53]: 1. Means the service to work those queues has not been started or the started queue is blocking that from being started

File: docs/Using RQ and RQScheduler.md (hunk 2 above)
Comment (Harriethw) [reply docs/Using RQ and RQScheduler.md:49]: feel free to add to the branch, or add a suggestion (cmd+g) that can be committed

File: docs/Using RQ and RQScheduler.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+The core service for facilitating RQ exported information is [RQ exporter](https://github.com/mdawar/rq-exporter). This will run on localhost:9726 to facilitate any RQ monitoring as data source. The configuration for the environment variables can be overridden, see more information at [Environment variable configuration](https://github.com/mdawar/rq-exporter#configuration). The datasource will be setup through _Prometheus_.
+
+### Environment
+
+**NOTE**: If you have a new or flushed docker system make sure you run `docker-compose -f docker-compose.yml -f docker-compose-rq-monitor.yml up --build` to build all the images or simply run `docker compose up --build` to build the core dependencies.
+
+- Starting all the services `make start-rq-mon`
+- Stopping all the services `make stop-rq-mon`
+
+## Troubleshooting
```
Comment (Harriethw) [inline docs/Using RQ and RQScheduler.md:50]: todo: add info about cron jobs needing to be cancelled

File: docs/Using RQ and RQScheduler.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+### Generic concepts to understand:
+- Workers
+    - These are services (when running on GovPaaS) or containers (when running locally on Docker). They loop through their assigned queues looking for jobs to process. They are started from the scripts `long-running-worker.py` and `short-running-worker.py`.
+- Queues
+    - These are just like real life queues, identified by unique strings (e.g. ‘long-running’). Jobs can be ‘enqueued’  on them, to be run asynchronously in order.
+- Jobs
+    - A job represents a function to be called by the worker.
+
+### Datahub specific code:
```
Comment (Harriethw) [inline docs/Using RQ and RQScheduler.md:17]: todo: add sequence diagram

**Review by marijnkampf — APPROVED** [review]

**Review by vfarah-if — APPROVED** [review]

**Pull request merged**